rxbindings/src/test/java/com/amplifyframework/rx/RxApiBindingTest.java

```diff
@@ -381,4 +378,16 @@ public void httpPostEmitsResult() {
         verify(delegate)
             .post(eq(options), anyConsumer(), anyConsumer());
     }
+
+    private static <R> GraphQLRequest<R> createMockMutationRequest(Class<R> responseType) {
+        return new GraphQLRequest<R>("", responseType, null);
+    }
+
+    private static <R> GraphQLRequest<Iterable<R>> createMockListRequest(Class<R> responseType) {
```

Comment:
Hmm..

R represents the `data` on a `GraphQLResponse`
T represents the `Model` type (or actually it looks like you've renamed most of these to `M` in this PR, which makes more sense than T anyway.

So, this should either return a `GraphQLRequest<R>` or a `GraphQLRequest<Iterable<T>>`, NOT a `GraphQLRequest<Iterable<R>>`

Reply:
updated everything here to `T`